documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationCustom.adoc

```diff
@@ -0,0 +1,45 @@
+To use custom authorization in Strimzi, you can configure your own `Authorizer` plugin to define Access Control Lists (ACLs).
```

Comment:
Can we also (somewhere here) add some explanation that users need to build their custom image and add the custom authorizer JARs to the classpath (`/opt/kafka/libs`)? Otherwise some might expect that it would be magically added just by using the API in the Kafka CR ;-).

Reply:
how about adding the below text? @scholzj @PaulRMellor 

For Strimzi to instantiate and configure the custom Authorizer class, apart from the additions to the custom resource as shown above, the JAR file containing the custom Authorizer class along with its dependencies must be available on the classpath of Kafka broker.

Strimzi maven build process provides a mechanism to add these custom third-party libraries to the generated Kafka broker container image by adding them as dependencies in the `pom.xml` file under `docker-images/kafka/kafka-thirdparty-libs` directory. Under this directory there are different folders for different Kafka versions, choose the folder that fits your needs. Before `pom.xml` is modified the third-party library must be available in a Maven repository, and that maven repository is accessible to the Strimzi build process.

Reply:
That sounds good to me! 👍